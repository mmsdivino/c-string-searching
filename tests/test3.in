A bola foi chutada!
A bola SERIA ARREMETIDA.
