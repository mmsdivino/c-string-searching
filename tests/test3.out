a 1, 2
bola 1, 2
foi fbi
chutada chuted
SERIA ceria
ARREMETIDA amreeta
